packages/jsii-pacmak/lib/targets/golang/types/enum.ts

```diff
@@ -1,27 +1,33 @@
 import { CodeMaker } from 'codemaker';
 import { EnumType } from 'jsii-reflect';
 import { GoType } from './go-type';
+import { Module } from '../module';
 
+// TODO: This whole thing
```

Comment:
See https://github.com/aws/jsii/pull/1892#discussion_r470880967

Reply:
That was a remnant from before we were emitting anything 😆